EducationPlannerBC/react-jsonschema-form
# Add trim call to TextareaWidget and BaseInput's onBlur event.target.value

### Reasons for making this change

Fixes: https://portal.bccampus.ca/jira/browse/EPUI-94

**Testing**
* Update package.json to use branch patch/epui-94 and re-install react-jsonschema-form
* Create an application for Capilano and open the Additional Information form.
* Select 'yes' for the field 'HAS YOUR EDUCATION BEEN INTERRUPTED FOR LONGER THAN SIX MONTHS?'
* Enter only spaces into the field 'PROVIDE A BRIEF OUTLINE OF YOUR ACTIVITIES DURING THIS PERIOD.'
* 'blur' away from this field should result in a validation error and the user cannot submit the form with out a valid string
* The middleware should no longer receive empty values in the formData for the EdInterruptAct field

- Added a checkboxes sample to 'simple' playgorund

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've checked the rendering of the Markdown text I've added
  - [ ] If I'm adding a new section, I've updated the Table of Content
* [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature
